dex-it/src/test/scala/com/wavesplatform/it/sync/api/ws/WsAddressStreamTestSuite.scala

```diff
@@ -323,27 +327,39 @@ class WsAddressStreamTestSuite extends WsSuiteBase with TableDrivenPropertyCheck
         wsc.close()
       }
 
-      "NTF asset issued " - {
+      "NTF asset" - {
+
+        def validateBalances(wsc: WsConnection, wb: WsBalances, a: KeyPair, nft: Boolean = true): Unit = {
```

Comment:
Suggest to use the better name `hasNft` instead of `nft`.

Reply:
ok